bindings/python/pinocchio/__init__.py

```diff
@@ -13,3 +13,9 @@
 from .libpinocchio_pywrap import *
 from .deprecated import *
 from .shortcuts import *
+
+try:
+  import hppfcl as fcl
+  def WITH_FCL_BINDINGS(): return True
```

Comment:
A constant would not work ?

```python
WITH_FCL_BINDINGS = True
```

Also, as a general style rule, you should avoid putting multiple statements on one line.

Reply:
Yes, shoud work too.

Reply:
Doen.